app/controllers/solidus_stripe/webhooks_controller.rb

```diff
@@ -5,24 +5,33 @@
 
 module SolidusStripe
   class WebhooksController < Spree::BaseController
-    SIGNATURE_HEADER = "HTTP_STRIPE_SIGNATURE"
+    before_action :load_payment_method
 
     skip_before_action :verify_authenticity_token, only: :create
 
     respond_to :json
 
+    rescue_from Webhook::Event::Error do |error|
+      logger.error(error.inspect)
```

Comment:
If we move this to a method instead of a block, we leave the possibility to people to change the behavior in case it's needed. Not blocking!